models/maisi_ct_generative/configs/inference.json

```diff
@@ -10,41 +10,40 @@
     "create_output_dir": "$Path(@output_dir).mkdir(exist_ok=True)",
     "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
     "trained_autoencoder_path": "$@model_dir + '/autoencoder_epoch273.pt'",
-    "trained_diffusion_path": "$@model_dir + '/input_unet3d_data-all_steps1000size512ddpm_random_current_inputx_v1.pt'",
-    "trained_controlnet_path": "$@model_dir + '/controlnet-20datasets-e20wl100fold0bc_noi_dia_fsize_current.pt'",
+    "trained_diffusion_path": "$@model_dir + '/diff_unet_ckpt_epoch19350.pt'",
+    "trained_controlnet_path": "$@model_dir + '/controlnet_current.pt'",
```

Comment:
I know that this is WIP. But could you please nail down the final version of the model name for the diffusion model and controlnet model to exclude any version number or epoch number so that users can easily fine-tune this bundle?

Thanks
Bin

Reply:
Thank you. It is not WIP. The controlnet_epoch208_v9.pt is the final version

Reply:
Sorry for the misleading. I mean rename the model from `controlnet_epoch208_v9.pt` to a name like `controlnet.pt`. I suppose this one would also be mentioned in the fine-tuning tutorial. Then users may fine-tuning their own model with this bundle. I think it would not be a good experience if users have to rename their fine-tuned model with name like `controlnet_epoch208_v9.pt`.

Reply:
Sure. Please feel free to go ahead and modify it to the name that sounds better. Thank you!